src/Fmt_odoc.ml

```diff
@@ -243,7 +252,8 @@ let fmt_block_element c = function
         | None -> fmt ""
       in
       hovbox 0
-        (wrap "{" "}" (str lvl $ lbl $ fmt "@ " $ fmt_inline_elements elems))
+        (wrap "{" "}"
+           (str lvl $ lbl $ fmt_inline_elements ~space:true elems))
```

Comment:
This space is not optional, is this to handle the case when `elems` is empty or only spaces ? In that case, I think that should be handled upfront and not with a flag.